english/eat-tacos/eat-delicious-tacos

```diff
@@ -0,0 +1,7 @@
+You found a place in the street, you were still hungry and you didn't have a lot of money. Then you decided to eat in that place.
+The only plate that they serve is tacos, you are kind of worried because the place doesn't look clean.
+However, it is your only option and you decided to give them a try.
+After the first bite, you feel like you are dead those tacos took you to
+
+[heaven](heaven/heaven.md)
+[the bathroom](bathdroom/bathroom.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.